ete3/tools/ete.py

```diff
@@ -94,9 +97,35 @@ def main():
 def _main(arguments):
     if len(arguments) > 1:
         subcommand = arguments[1]
-        if  subcommand == "version":
+        if subcommand == "version":
             from .. import __version__
-            print(__version__)
+
+            _version = __version__
+
+            try:
+                # If on a git repository and tags are available
+                # Use a tag based code (e.g. 3.1.1b2-8-gb2d12f4)
+                p = Popen(["git", "describe", "--tags"], stdout=PIPE, stderr=PIPE)
+                out, err = p.communicate()
+            except OSError as e:
+                if e.errno == errno.ENOENT:
+                    # Git not installed
+                    pass
+                else:
+                    raise
```

Comment:
What scenario could take us here? Only if git is installed but "git describes" exits badly? Then maybe better set __version__+"-unknown" or similar?